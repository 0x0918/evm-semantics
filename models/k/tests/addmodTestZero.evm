account:
-   id: 1
-   balance: 40
-   program: PUSH 0 ; PUSH 3 ; PUSH 4; ADDMOD
-   storage: 

.Accounts

START 1
