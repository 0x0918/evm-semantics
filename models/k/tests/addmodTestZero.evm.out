<T> <k> . </k> <accountID> 1 </accountID> <pc> 4 </pc> <wordStack> 0 : .WordStack </wordStack> <localMem> .Map </localMem> <callStack> .CallStack </callStack> <accounts> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 0 ) 1 |-> ( PUSH 3 ) 2 |-> ( PUSH 4 ) 3 |-> ADDMOD </program> <storage> .Map </storage> <balance> 40 </balance> </account> </accounts> </T>
